charts/argo-cd/templates/_helpers.tpl

```diff
@@ -159,12 +159,14 @@ Create the name of the notifications service account to use
 {{- end -}}
 
 {{/*
-Argo Configuration Preset Values (Incluenced by Values configuration)
+Argo Configuration Preset Values (Influenced by Values configuration)
 */}}
 {{- define "argo-cd.config.cm.presets" -}}
 {{- $presets := dict -}}
 {{- $_ := set $presets "url" (printf "https://%s" .Values.global.domain) -}}
+{{- if and (hasKey .Values.configs.cm "statusbadge.enabled") (index .Values.configs.cm "statusbadge.enabled" | eq true) -}}
```

Comment:
nits: It's better to add this flag on values.yaml explicitly? 🙋 
https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/values.yaml#L155

Reply:
@yu-croco Hi, you mean to add statusbadge.enabled to values, avoid hasKey and just check for true? 
That sounds good, I can push this when I get some time later today.

Reply:
@yu-croco Hi, I've incorporated your suggestion into the changes, updated the docs and cleaned up extraneous commits to tidy everything up. Please let me know if anything else comes to mind.